docs/sink-connector-config-options.rst

```diff
@@ -55,6 +108,14 @@ Batching
 Delivery
 ^^^^^^^^
 
+``kafka.retry.backoff.ms``
+  The retry backoff in milliseconds. This config is used to notify Kafka Connect to retry delivering a message batch or performing recovery in case of transient exceptions. Maximum value is 86400000 (24 hours).
```

Comment:
```suggestion
  The retry backoff in milliseconds. This config is used to notify Kafka Connect to retry delivering a message batch or performing recovery in case of transient failures.
```